frontend/src/main/web/lib/constants/ActionTypes.js

```diff
@@ -0,0 +1,25 @@
+import keymirror from 'keymirror';
+
+var UserMatrixActionTypes = keymirror({
+  DATE_RANGE_UPDATE: null,
+  CONTENT_STATE_UPDATE: null,
+  DAY_SELECTED: null
+});
+
+var GlossaryActionTypes = keymirror({
+  TRANS_LOCALE_SELECTED: null,
+  INSERT_GLOSSARY: null,
+  UPDATE_GLOSSARY: null,
+  DELETE_GLOSSARY: null,
+  UPDATE_FILTER: null,
+  LOAD_GLOSSARY: null,
+  UPDATE_SORT_ORDER: null,
+  UPLOAD_FILE: null,
+  UPDATE_ENTRY_FIELD: null,
+  UPDATE_COMMENT: null,
+  UPDATE_FOCUSED_ROW: null,
+  RESET_ENTRY: null
+});
+
+exports.UserMatrixActionTypes = UserMatrixActionTypes;
+exports.GlossaryActionTypes = GlossaryActionTypes;
```

Comment:
Proper es6 export syntax for these is:

```
export UserMatrixActionTypes, GlossaryActionTypes
```

it exports them with the name they already have, so no need to repeat the name